source/contribute/maintainer.rst

```diff
@@ -59,16 +67,167 @@ Optionally, add any additional contributor/maintainer forks, e.g.:
 
     $ git remote add romanb git://github.com/romanb/doctrine2.git
 
+You can also fetch all open pull requests via `git fetch` if you
+edit your ``.git/config`` as following:
+
+... code-block:: console
+
+    [remote "doctrine"]
+        url = git@github.com:doctrine/doctrine2.git
+        fetch = +refs/heads/*:refs/remotes/doctrine/*
+        # add this:
+        fetch = +refs/pull/*/head:refs/remotes/doctrine/pr/*
+
+Distinguishing features, bugs and critical fixes
+------------------------------------------------
+
+The primary role of a maintainer, besides being also a contributor,
+is to sort incoming proposals by their category:
```

Comment:
As part of this standardization, we SHOULD sync labels across all repositories and also their colors (at least all listed below). Currently it's random.